dataland-frontend/tests/e2e/specs/company-metadata/CompaniesOnlySearch.ts

```diff
@@ -24,4 +37,23 @@ describe("As a user, I want to be able to search companies existing on Dataland"
       .click({ force: true });
     verifyCompanySearchResultTable();
   });
+
+  it("Company Name Input field exists and works", () => {
+    const inputValue = companiesWithEuTaxonomyDataForNonFinancials[0].companyInformation.companyName;
+    cy.visitAndCheckAppMount("/companies-only-search");
+    cy.get("input[name=companyName]")
+      .should("not.be.disabled")
+      .type(inputValue, { force: true })
+      .should("have.value", inputValue);
+    cy.intercept("**/api/companies*").as("retrieveCompany");
+    cy.get("button[name=getCompanies]").click();
+    cy.wait("@retrieveCompany", { timeout: 60 * 1000 }).then(() => {
+      cy.get("td").contains(companiesWithEuTaxonomyDataForNonFinancials[0].companyInformation.companyName);
+    });
+  });
+
+  it("Show all companies button exists", () => {
```

Comment:
Dieser Test ist bereits in dem ersten Test enthalten. Ich finde es aber gut es in zwei Tests aufzuteilen. Ich würde dann aber die redundaten Teile im ersten Test entfernen.

Reply:
Removed the _.should_ checks of the button in the first test, since these are covered in the latter one; The _click_-call is however still necessery